pkg/service/manifest/storage/storage.go

```diff
@@ -26,7 +26,7 @@ const (
 
 type StoredManifest struct {
 	ID          string                      `json:"id"`
-	Issuer      string                      `json:"issuer"`
+	IssuerDID   string                      `json:"issuerDid"`
```

Comment:
Was this intentional?

Reply:
this one was to be consistent with the others